README.md

```diff
@@ -38,10 +38,13 @@ Publish contracts
 
 Options:
   --url             Url to judge-d instance                  [string] [required]
-  --pactsDir        Path to directory with pacts             [string] [required]
   --serviceName     Service name                             [string] [required]
   --serviceVersion  Service version                          [string] [required]
 
+  Required one of:
+  --pactsDir        Path to directory with pacts             [string]
+  --swaggerDir      Path to directory with swagger file      [string]
```

Comment:
Is it possible to have more than 1 swagger file generated for a given service? I believe that answer is no, and if it is really the case then we should rather be passing  `--swaggerFile`  instead of `--swaggerDir`